bazelisk.go

```diff
@@ -566,6 +579,41 @@ func downloadBazel(fork string, version string, isCommit bool, directory string)
 	return destinationPath, nil
 }
 
+func copyFile(src, dst string, perm os.FileMode) (err error) {
+	data, err := ioutil.ReadFile(src)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(dst, data, perm)
```

Comment:
I would prefer to use `io.Copy` here instead of `ioutil.ReadFile` and `ioutil.WriteFile`, so that we don't read the entire file into RAM first, as the Bazel binary can be quite large.